### PR TITLE
Add tier2 role

### PR DESCRIPTION
**Pull request by iamkirkbater**
### What type of PR is this?
Feature

### What this PR does / why we need it?
Adds MCS Tier Two support role to clusters so that they can use backplane to login. Direct Copy of the CEE role at this time. Milestone Two of the MCS Enablement Epic may give them additional permissions, but no extra permissions are given at this time.

### Which Jira/Github issue(s) this PR fixes?

_Fixes https://issues.redhat.com/browse/OSD-25352_

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by iamkirkbater** [comment]
/approve cancel

I believe we need Architect review for permissions so I want to remove the approval so this doesn't accidentally get LGTM'd

File: deploy/backplane/mcs-tier-two/hypershift/service-cluster/10-mcs-tier-two-service-cluster-cluster.ClusterRole.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - list
+  - watch 
+- apiGroups:
+  - policy.open-cluster-management.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
```
Comment (jewzaam) [inline deploy/backplane/mcs-tier-two/hypershift/service-cluster/10-mcs-tier-two-service-cluster-cluster.ClusterRole.yml:61]:
```suggestion
```

Duplicate.

File: deploy/backplane/mcs-tier-two/hypershift/service-cluster/20-mcs-tier-two.SubjectPermission.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mcs-tier-two-service-cluster
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - backplane-mcs-tier-two-service-cluster-cluster
+  - view
```
Comment (jewzaam) [inline deploy/backplane/mcs-tier-two/hypershift/service-cluster/20-mcs-tier-two.SubjectPermission.yml:9]:
```suggestion
  - backplane-readers-cluster
```

Inconsistent with docs.

File: deploy/backplane/mcs-tier-two/hypershift/management-cluster/20-mcs-tier-two.SubjectPermission.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mcs-tier-two-management-cluster
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - backplane-mcs-tier-two-management-cluster-cluster
+  - view
```
Comment (jewzaam) [inline deploy/backplane/mcs-tier-two/hypershift/management-cluster/20-mcs-tier-two.SubjectPermission.yml:9]:
```suggestion
  - backplane-readers-cluster
```

Inconsistent with docs.

**Review by jewzaam — CHANGES_REQUESTED** [review]
Not mentioned in docs:
* pcap-collector
* must-gather-operator
* CMO portforward

Are these necessary or an artifact of copy/paste?  Either remove or update docs please.

**Review by jewzaam — APPROVED** [review]
/approve

**Pull request merged**